tests/benchmark/README.md

```diff
@@ -0,0 +1,126 @@
+# Benchmarking
+
+This section contains tooling which allow us to evaluate how long astro tasks take and how much computation resources they consume.
+
+We are using some existing datasets to this analysis, and it can be easily customised.
+
+
+## Configuration
+
+Create a file named `config.json` based on the datasets and databases you would like to benchmark.
+
+There is an example `./config-sample.json`.
+
+Example of content:
+```
+{
+  "databases": [
+    {
+      "name": "postgres",
+      "params": {
+        "database": "postgres",
+        "conn_id": "postgres_conn",
+        "schema": "postgres"
+      }
+    },
+    {
+      "name": "bigquery",
+      "params": {
+        "database": "bigquery",
+        "conn_id": "bigquery"
+      }
+    }
+  ],
+  "datasets": [
+    {
+        "name": "few_kb",
+        "size": "45K",
+        "path":"/tmp/covid_overview.csv",
```

Comment:
Improve this - allow reproducibility by either adding a command to download these datasets or add them to a public GCS folder.